R/ok.R

```diff
@@ -50,8 +50,15 @@ ok.character <- function(x, status = 200L, info = TRUE, ...) {
 ok.HttpClient <- function(x, status = 200L, info = TRUE, ...) {
   assert(info, "logical")
   assert(status, "integer")
-  find_status <- tryCatch(fauxpas::find_error_class(status), 
-    error = function(e) e)
+  
+  if (!requireNamespace("fauxpas", quietly = TRUE)) {
```

Comment:
woops, sorry, I meant to just use httpcode, so can get rid of the if/else and just use httpcode, sound good?

Reply:
this makes testing easier as well

Reply:
Aaah it makes sense! Done!

Reply:
thanks